### PR TITLE
Use an embed override for timer service

### LINKED ISSUE #22733: Cleanup ampdoc-fie experiment
See launch in #22734

TODO for cleanup:

- [x] Remove `FriendlyIframeEmbedLegacy`
- [x] `FriendlyIframeEmbed` make `ampdoc` non-nullable and update the code accordingly.
- [x] cleanup ampdoc service lookup
- [x] Remove `EmbeddableService.installInEmbedWindow` interface.
- [x] Remove `installServiceInEmbedScope` function.
- [x] Remove `opt_win`, `opt_root`, etc arguments from `Bind`, `AmpGwdRuntimeService`, `HiddenObserver`, `Action`, `StandardActions`, `Navigation`, `Url`, and other embeddable services. (See #22493 for some examples).
- [x] Remove the `getExistingServiceForDocInEmbedScope`.
- [x] Remove the `installStylesLegacy`.

### PR DESCRIPTION
**Pull request by dvoytenko**
Closes #22733.

### CODE REVIEW

**Review by jridgewell — APPROVED** [review]

File: extensions/amp-a4a/0.1/amp-a4a.js [hunk 1]
```diff
@@ -1848,9 +1848,7 @@ export class AmpA4A extends AMP.BaseElement {
   preinstallCallback_(embedWin, ampdoc) {
     const parentAmpdoc = this.getAmpDoc();
     installUrlReplacementsForEmbed(
-      // TODO(#22733): Cleanup `parentAmpdoc` once ampdoc-fie is launched.
-      ampdoc || parentAmpdoc,
-      embedWin,
+      ampdoc,
       new A4AVariableSource(parentAmpdoc, embedWin)
```
Comment (samouri) [inline extensions/amp-a4a/0.1/amp-a4a.js:1852]: Is this related to the rest of this PR?

Comment (dvoytenko) [reply extensions/amp-a4a/0.1/amp-a4a.js:1852]: Not the `ampdoc || parentAmpdoc` specifically. But since I was here, and it was the same overall tracking issue, I went ahead and fixed.
